packages/plugins/1password/src/plugin.ts

```diff
@@ -0,0 +1,218 @@
+import { Resolver } from 'varlock/plugin-lib';
+
+import { createDeferredPromise, DeferredPromise } from '@env-spec/utils/defer';
+import { Client, createClient } from '@1password/sdk';
+import { opCliRead } from './cli-helper';
+
+const { ValidationError, SchemaError, ResolutionError } = plugin.ERRORS;
+
+const PLUGIN_VERSION = plugin.version;
+const OP_ICON = 'simple-icons:1password';
+
+plugin.name = '1pass';
```

Comment:
plugin is injected globally... its a bit awkward, but the best way I've found so far to inject in memory objects into dynamically imported code. Otherwise we'd end up re-bundling copies of our classes and functions into the plugin, and instanceof checks would not work.